foghorn-edge/scripts/init-folder-structure.sh

```diff
@@ -0,0 +1,15 @@
+
+mkdir -p /fh_vol/fh/certs/ca
```

Comment:
Following commands are execute in root user ? OpenNESS 21.03 support non-root user execution.